Examples/Nodify.Playground/EditorSettings.cs

```diff
@@ -7,9 +11,268 @@ public enum ConnectionStyle
         Circuit
     }
 
-    public class EditorSettings : ObservableObject
+    public class EditorSettings : ObservableObject, IEnumerable<ISettingViewModel>
```

Comment:
No need to implement IEnumerable if you don't bind directly to the instance in the ItemsSource.